pkg/operation/botanist/etcdencryption.go

```diff
@@ -123,25 +237,7 @@ func (b *Botanist) createOrUpdateEncryptionConfiguration(ctx context.Context) (*
 
 		return encryptionconfiguration.UpdateSecret(secret, conf)
 	})
-	if err != nil {
-		return nil, err
-	}
 
-	return secret, err
-}
-
-func (b *Botanist) syncEncryptionConfigurationToGarden(ctx context.Context, controlPlaneSecret *corev1.Secret) error {
```

Comment:
Good point, so we can clean up the `<shootname>.etcd-encryption` secrets in the project namespaces.

Reply:
They would be removed automatically as they have shoot owner references.
However, I could add logic to clean up our shooted seeds and other shoots which won't be deleted for a long time.